CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- semantics for CreateContextStateWithAssociationAndValidators manipulation
```

Comment:
this change is already included in the changed section. The former one can be removed.

Reply:
removed